certified-connectors/Mavim iMprove/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4478 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "Mavim-iMprove",
```

Comment:
@RalphNoordanus I would suggest you name your connector as `Mavim` , instead of 'Mavim-iMprove` as per the connector's description.

Reply:
@sriyen-msft We are using Mavim iMprove because we will have multiple connectors to be pushed to the store. Mavim Process Mining; Mavim Admin; ...